ls1intum/Artemis
# [Enhancement] - Make query for quiz import more performant

### Checklist

### Motivation and Context


Courses and exams of quiz questions were fetched in two separate database calls which was redundant

### Description

- The courses and exams which contain quiz questions are now fetched in one database call
- Also mentioned in the documentation that nested queries should be prevented.

### Steps for Testing


1. Log in to Artemis
2. Import course and exam quiz questions for a new quiz exercise in a course
3. Import course and exam quiz questions for a new quiz task in an exam
4. Make sure everything works (PR only changes performance, no new functionality)
